test/e2e-go/cli/goal/expect/goalCmdFlagsTest.exp

```diff
@@ -22,8 +22,8 @@ proc TestGoalCommandLineFlags { CMD EXPECTED_RE } {
 if { [catch {
     puts "Part 1: Check --validrounds and --lastvalid options combination for 'goal asset' and 'goal account"
 
-    TestGoalCommandLineFlags "goal asset create --validrounds 0 --creator ABC --total 100" ".*can not be zero.*"
-    TestGoalCommandLineFlags "goal asset create --validrounds 1 --lastvalid 1 --creator ABC --total 100" "Only one of .* can be specified"
+    TestGoalCommandLineFlags "goal asset create --decimals 0 --validrounds 0 --creator ABC --total 100" ".*can not be zero.*"
```

Comment:
isn't 0 the default? Consider testing some other numbers. (and w/o the flag, to make sure the default is actually being set)

Reply:
done -- yeah the flags were left over from when I made the decimals field mandatory (it's not now)